plugins/modules/ovirt_disk.py

```diff
@@ -186,6 +186,11 @@
             - I(True) if the disk should be activated.
             - When creating disk of virtual machine it is set to I(True).
         type: bool
+    use_proxy:
+        description:
+            - "Use Image I/O proxy when uploading or downloading disk image. Set
+              this to I(True) if you cannot directly connect to the oVirt node."
```

Comment:
Looks good, but we should also warn that using the proxy is less efficient and not recommended.

In 4.4 the proxy can be 10 times faster than the proxy in 4.3, but it is still 2 times slower compared with
direct upload. These numbers are not official since scale team never test this, but it illustrates the difference.

Reply:
Personally I don't care how fast this upload is. The main point is that it works. I had a playbook using this module against an existing oVirt 4.3 which worked fine. Then I found it to be broken by oVirt 4.4 so I fixed the module and thought I could share it. If there is a better way to do it, go ahead with your solution.